<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog2</class>
 <widget class="QDialog" name="Dialog2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>316</width>
    <height>279</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Адрес сервера</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Порт</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Пароль</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="addressLine">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>281</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="portLine">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="passwordLine">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>281</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="setdefPB">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>240</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>По умолчанию</string>
   </property>
  </widget>
  <widget class="QPushButton" name="connectPB">
   <property name="geometry">
    <rect>
     <x>124</x>
     <y>240</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Подключиться</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelPB">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>240</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Отмена</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
